frontend/src/components/featured-post-card.tsx

```diff
@@ -9,17 +9,17 @@ export default function FeaturedPostCard({ post }: { post: Post }) {
   const slug = createSlug(post.title);
   return (
     <div
-      className="flex h-48 cursor-pointer gap-2 rounded-lg bg-light dark:bg-dark-card"
+      className="flex flex-col sm:flex-row h-auto sm:h-48 cursor-pointer gap-2 rounded-lg bg-light dark:bg-dark-card"
       onClick={() => navigate(`/details-page/${slug}/${post._id}`, { state: { post } })}
     >
-      <div className="w-1/3">
+      <div className="w-full sm:w-1/3">
         <img
```

Comment:
perfect :), Can we do similar to https://github.com/krishnaacharyaa/wanderlust/blob/main/frontend/src/components/blog-feed.tsx, 

Here we are doing the same thing like `md:w-2/3` and `md: w-1/3`, Can we change all the places where md is used to use sm instead. That would help us maintain the consistency.

Reply:
I would like to take up refactoring as a separate issue.